CMakeLists.txt

```diff
@@ -2398,7 +2398,10 @@ if(BROADCAST)
   elseif(Shoutidjc_FOUND AND Shoutidjc_VERSION VERSION_LESS 2.4.6)
       message(STATUS "Installed libshout version: ${Shout_VERSION} is suffering from bug lp1913579")
   endif()
-  if(NOT Shsoutidjc_FOUND OR Shoutidjc_VERSION VERSION_LESS 2.4.6)
+  if(NOT Shoutidjc_FOUND OR Shoutidjc_VERSION VERSION_LESS 2.4.6)
+    if(UNIX AND NOT APPLE)
+      find_package(OpenSSL COMPONENTS SSL REQUIRED)
+    endif()
```

Comment:
I think this is the wrong place here.
In /mixxx/lib/libshout-idjc/CMakeLists.txt we have: 

```
# Optional system dependencies
find_package(OpenSSL)
if(TARGET OpenSSL::SSL)
    target_compile_definitions(${PROJECT_NAME} PRIVATE HAVE_OPENSSL)
    target_link_libraries(${PROJECT_NAME} PRIVATE OpenSSL::SSL OpenSSL::Crypto)
endif()
```

Original tls.c was excluded by configure, if openssl was not found. 
We should either make openssl mandatory here, or restore the optional logic for tls.c

Reply:
Ok. then let's make OpenSSL mandatory. Randomly failing to connect to servers because they require TLS (which is probably the vast majority today) would be unexpected.